src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/_scaffold.scss

```diff
@@ -1,5 +1,6 @@
 body{
```

Comment:
I want to keep scaffold.scss just for layout things so let's work through what is repeatable here and what we want to move into other .scss partials.